scripts/release-all.js

```diff
@@ -1,16 +1,14 @@
 const release = require('./release.cjs');
 
-(async () => {
```

Comment:
Not a part of this PR specifically, but `release()` is sync… shouldn’t it just be run synchronously?

Reply:
works for me! I think at one point it may have been async? /shrug